package.json

```diff
@@ -6,7 +6,7 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
     "build": "docusaurus build",
-    "build.ci": "docusaurus build && mv build www",
+    "build.ci": "docusaurus build",
```

Comment:
Does anyone know why this `mv` command is here? I suspect it's from the initial docusaurus migration, but I'm not 100% sure

Reply:
Also, it's there as legacy code from the stencil site days

Reply:
Interestingly, I tried to remove it, but CI failed trying to find that command. I couldn't find `build.ci` anywhere else in the codebase, but couldn't (and assumed this was just a convention that Vercel used when `vercel build` is run) - do you know if that's actually the case?

Reply:
Oh, nvm. It seems to work now. Unsure what I was doing wrong earlier 🤔

Reply:
Hmmm other deploys work now....maybe it was just a transient Vercel issue? Might as well still rm this command, gonna keep this PR up

Reply:
That was me changing the vercel setup to the docusaurus instead of stencil preset for purposes of local development